src/transport/virtiofs/mod.rs

```diff
@@ -437,7 +449,9 @@ mod async_io {
     }
 }
 
-#[cfg(test)]
+/// Disabled since vm-virtio doesn't export any DescriptorChain constructors.
```

Comment:
I'm trying to export DescriptorChain::new()

Reply:
Great. We can re-enable these tests once `DescriptorChain::new()` is exported.